src/components/BookForm.js

```diff
@@ -19,10 +19,13 @@ function BookForm() {
   }
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input type="text" placeholder="Title" value={title} required onChange={(ev) => setTitle(ev.target.value)} />
-      <input type="text" placeholder="Author" value={author} required onChange={(ev) => setAuthor(ev.target.value)} />
-      <button type="submit">Add Book</button>
+    <form className="form" onSubmit={handleSubmit} action="">
+      <h2>Add new book</h2>
```

Comment:
- [ ] In the same light, the ADD NEW BOOK text and Book title input form element should align with the section above it as you see in the screenshot below.
![Screen Shot 2023-04-15 at 3 47 43 AM](https://user-images.githubusercontent.com/62849872/232209501-08e769ae-6bb3-419b-abbb-d79048af0f5c.png)

Reply:
thanks for your kind suggestions @t-yanick , I will do my best to implement all of these changes.